modules/git/git.go

```diff
@@ -224,6 +224,29 @@ func syncGitConfig() (err error) {
 		return err
 	}
 
+	if setting.Git.Reflog.Enabled {
```

Comment:
Is there an easy way to check what versions of Git support this new feature? A testing matrix or anything? Reflogs are pretty old so I was having trouble figuring out what the oldest version of Git that supports them is.

Reply:
I just had a look at the git repo with a `git blame`, I've been able to trace reflogs back to Git 1.5.
As the minimum supported Git version is 2.0 there is nothing to worry about in this case.